.circleci/config.yml

```diff
@@ -214,6 +223,18 @@ commands:
     description: >-
       skip job unless required
     steps:
+      - run:
```

Comment:
I do not see this step in Android CircleCI job. Is this not relevant in Android?

Reply:
this feature for android was already checked in  before